TypeDBConsole.java

```diff
@@ -444,6 +447,10 @@ private TypeDBClient createTypeDBClient(CLIOptions options) {
                 String optCluster = options.cluster();
                 if (optCluster != null) {
                     client = TypeDB.clusterClient(set(optCluster.split(",")), createTypeDBCredential(options));
+                    long expiryDays = client.asCluster().users().get(options.username).expiryDays();
+                    if (expiryDays <= 7) {
```

Comment:
This should be a constant `PASSWORD_EXPIRY_WARN_DAYS` somewhere

Reply:
Agreed, implemented.